.github/workflows/edge-home-orchestration-go.yml

```diff
@@ -8,5 +8,20 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Setup Golang
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.15.6'
+
+      - name: Set env vars
+        run: |
+          go get github.com/axw/gocov/gocov
+          echo "$HOME/go/bin" >> $GITHUB_PATH
+  
       - name: Build the project
         run: ./build.sh
+
+      - name: Run the Test Suite
+        run: |
+          docker stop edge-orchestration
+          gocov test -v $(go list ./src/... | grep -v src/controller/discoverymgr | grep -v src/controller/servicemgr/executor/containerexecutor | grep -v src/orchestrationapi) -coverprofile=/dev/null
```

Comment:
How about omitting -v since it doesn't need full logs?

Reply:
Done!